core/arch/x86/x86.asm

```diff
@@ -1884,8 +1884,8 @@ GLOBAL_LABEL(cpuid_supported:)
         DECLARE_FUNC(our_cpuid)
 GLOBAL_LABEL(our_cpuid:)
         mov      REG_XAX, ARG1
-        mov      REG_XDX, ARG2
         mov      REG_XCX, ARG3
+        mov      REG_XDX, ARG2
```

Comment:
Looking through some others: dr_longjmp looks incorrect for Windows with ARG2 in rdx being clobbered.  Tweak that one?

Reply:
I'll look at this separately right afterwards.